pubs/config/spec.py

```diff
@@ -27,6 +27,10 @@
 # Which default extension to use when creating a note file.
 note_extension = string(default='txt')
 
+# How many authors to display when displaying a citation. If there are more
+# authors, only the first author is diplayed followed by 'et al.'.
+n_authors = integer(default=3)
+
```

Comment:
Because I did not read the full line I was initially under the impression that `n_authors` meant "we display n then put 'et al.'". Maybe that's just me but we could also write "Maximum number of authors..." or use `max_` in the variable name. I am not sure, though, what would be the clearest and it is really minor, so you can ignore this comment if you feel differently.

Reply:
Yes, `max_authors` is more correct than `n_authors`.